nntrainer/graph/network_graph.cpp

```diff
@@ -831,6 +832,58 @@ NetworkGraph::finalizeContext(const std::shared_ptr<LayerNode> &lnode,
   return outputs;
 }
 
+std::map<std::string, std::vector<unsigned int>> NetworkGraph::getLayerExecutionOrders(
+    const std::shared_ptr<LayerNode> &lnode) {
+  const GraphNode &gnode = *lnode.get();
+  auto init_context = lnode->getInitContext();
+  auto out_specs = init_context.getOutSpecs();
+  auto weight_specs= init_context.getWeightsSpec();
+  auto tensor_specs = init_context.getTensorsSpec();
+
+  std::map<std::string, std::vector<unsigned int>> exec_orders;
+
+  for (auto &spec : out_specs) {
+    const auto name = lnode->getName() + ":" + spec.variable_spec.name;
+    auto orders = tensor_manager->getTensorExecutionOrders(name, false);
+    exec_orders.insert({name, orders});
+    try {
+      auto orders2 = tensor_manager->getTensorExecutionOrders(name + ":grad", false);
```

Comment:
Wouldn't it be more intuitive to set orders to forward_orders and orders2 to gradient_orders?

Reply:
Looks better. thanks.